### ISSUE TITLE
Confirmed-commit support[Question]

### ISSUE THREAD

**Issue by ankit7gup** [CESNET/netopeer2#1025]
Hi Michal, 

Have a query regarding confirmed-commit capability support. Are we supporting confirmed-commit now? If no, Is there any plan in near future for the same.

Thanks!

Regards,
Ankit

**Comment by michalvasko** [comment]
No, ti is not supported but should be in a matter of days. We are just finalizing the PR, the functionality is finished and working.
